bin/get-metadata

```diff
@@ -1,17 +1,11 @@
 #!/usr/bin/env node
 'use strict';
 
-const argv = require('../lib/args')();
-const getMetadata = require('../components/metadata');
-const CLI = require('../lib/cli');
+const path = require('path');
+const { runAsync } = require('../lib/run');
 
-const logStream = process.stdout.isTTY ? process.stdout : process.stderr;
-const cli = new CLI(logStream);
+process.emitWarning('Please use `git-node metadata` instead', 'get-metadata');
```

Comment:
Can we not do this as i said before we want this to remain same, since most people are using this or used to it?